src/main/java/cucumber/runtime/formatter/SerenityReporter.java

```diff
@@ -330,11 +330,7 @@ private void handleTestStepFinished(TestStepFinished event) {
     }
 
     private void handleTestRunFinished(TestRunFinished event) {
-        if (examplesRunning) {
-            finishExample();
```

Comment:
I guess this is unreachable code as `exampleRunning` will be always `false` as `finishExample` method was already executed during `TestCaseFinished` event.